src/workspace/view/newworkspacemodel.cpp

```diff
@@ -143,10 +143,35 @@ QVariant NewWorkspaceModel::createWorkspace()
     }
 
     for (WorkspaceTag tag : importedTags) {
+        addStubData(newWorkspace, tag);
         for (AbstractDataPtr data : currentWorkspace->dataList(tag)) {
             newWorkspace->addData(data);
         }
     }
 
     return QVariant::fromValue(newWorkspace);
 }
+
+void NewWorkspaceModel::addStubData(IWorkspacePtr workspace, WorkspaceTag tag) const
```

Comment:
I think, better to place this code to the workspace implementation, not in this model

Reply:
decided to save tags instead of creating stubs